exercises/concept/strings-basic/pubspec.yaml

```diff
@@ -1,4 +1,4 @@
-name: 'strings'
+name: 'strings_basic'
```

Comment:
I cannot find this anywhere right now, but I am pretty certain that the concepts weren't supposed to be split into different chunks based on difficulty, e.g. `strings-basics` and `strings-advanced`. It was written somewhere in an old document a year ago...

Reply:
@angelikatyborska Thanks. I'll look that up. I believe I was basing this on one of the earlier concept exercises for a different project. But I'll double check if there's anything against this process.